packages/manager/modules/hub/src/components/support/controller.js

```diff
@@ -0,0 +1,49 @@
+import get from 'lodash/get';
+import { MAX_TICKETS_TO_DISPLAY } from './constants';
+import SupportTicket from './ticket.class';
+
+export default class ManagerHubSupportCtrl {
+  /* @ngInject */
+  constructor($http, $q, RedirectionService) {
+    this.$http = $http;
+    this.$q = $q;
+    this.RedirectionService = RedirectionService;
+  }
+
+  $onInit() {
+    this.isLoading = true;
+    return this.$q
+      .when(this.tickets ? this.tickets : this.fetchTickets())
+      .then((tickets) => {
+        if (tickets) {
+          this.tickets = tickets
+            .slice(0, MAX_TICKETS_TO_DISPLAY)
+            .map((ticket) => new SupportTicket(ticket));
+        }
+      })
```

Comment:
```suggestion
      .then((tickets = []) => {
          this.tickets = tickets
            .slice(0, MAX_TICKETS_TO_DISPLAY)
            .map((ticket) => new SupportTicket(ticket));
      })
```

Reply:
it's almost the same but would break if ticket is ```null``` i guess the correct way would be to check with ```if (Array.isArray(tickets) ```